index.html

```diff
@@ -22,17 +22,329 @@
+        </nav>
+    </nav>
+    
+    <nav class="container33">
+        <ul class="comp5">
+            <li class="rect5"><img src="imgformobile/Rectangle5.png" alt="computer"></li>
+        </ul>
+    </nav>
+    <nav class="container34">
+        <ul>
+            <li class="Proj5"><a href="#">Hello-World</a></li>
+        </ul>
+        <nav class="container35">
+            <ul class="data13">
+                <li class="Lang12"><a href="#">HTML/CSS</a></li>
+            </ul>
+        </nav>
+        <nav class="container36">
+            <ul class="data14">
+                <li class="Lang13"><a href="#">Ruby on </a></li>
+            </ul>
+        </nav>
+        <nav class="container37">
+            <ul class="data15">
+                <li class="Lang14"><a href="#">JavaScript</a></li>
+            </ul>
+        </nav>
+    
+        <nav class="container38">
+            <ul class="navlink6">
+                <li class="project5"><a href="#">See this Project</a></li>
+                <li class="arrow4"><a href="#"><img src="imgformobile/Union.png" alt="sign"></a></li>
+            </ul>
+        </nav>
+    </nav>
+    
+    <nav class="container39">
+        <ul class="comp6">
+            <li class="rect6"><img src="imgformobile/Rectangle6.png" alt="computer"></li>
+        </ul>
+    </nav>
+    <nav class="container40">
+        <ul>
+            <li class="Proj6"><a href="#">Hello-World</a></li>
+        </ul>
+        <nav class="container41">
+            <ul class="data16">
+                <li class="Lang15"><a href="#">HTML/CSS</a></li>
+            </ul>
+        </nav>
+        <nav class="container42">
+            <ul class="data17">
+                <li class="Lang16"><a href="#">Ruby on </a></li>
+            </ul>
+        </nav>
+        <nav class="container43">
+            <ul class="data18">
+                <li class="Lang17"><a href="#">JavaScript</a></li>
+            </ul>
+        </nav>
+    
+        <nav class="container44">
+            <ul class="navlink7">
+                <li class="project6"><a href="#">See this Project</a></li>
+                <li class="arrow5"><a href="#"><img src="imgformobile/Union.png" alt="sign"></a></li>
+            </ul>
+        </nav>
+    </nav>
+    
+
+
+
+    <div class="container45">
+        <h3>About</h3>
+    </div>
+    <div class="container46">
+        <p>Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works.
+    If you like what you see and have a project you need coded, don’t hesitate and contact me.</p>
+    </div>
+    <div class="container47">
+        <div class="CV">
+            <p>Get my CV</p>
         </div>
+    </div>
+    <nav>
+        <ul>
+            <li class="arrow6"><a href="#"><img src="imgformobile/Rectangle45.png" alt="sign"></a></li>
+            <li class="arrow7"><a href="#"><img src="imgformobile/Union1.png" alt="sign"></a></li>
+        </ul>
+    </nav>   
```

Comment:
- [ ] The button (e.g., "Get my CV") is a link `a` or `button` tag, and follow the button styling: Active, Hover and Pressed (see design), currently only the icon is clickable kindly make the text a link or button 👍🏿